specs/latest/2.0/index.html

```diff
@@ -4196,6 +4196,81 @@ <h3><code>deleteQuery</code> implicitly calls <code>endQuery</code> if the query
       point it's better to spec what we implemented.
     </div>
 
+    <h3>Required compressed texture formats</h3>
+    <p>
+        Implementations must support at least one suite of compressed texture formats.
+    </p>
+    <p>
+        <br>Implementations must support:
+        <ul>
+          <li><a href="https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_etc/">WEBGL_compressed_texture_etc</a> AND/OR
+          <li>(
+            <ul>
+              <li><a href="https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_s3tc/">WEBGL_compressed_texture_s3tc</a> AND
+              <li><a href="https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_s3tc_srgb/">WEBGL_compressed_texture_s3tc_srgb</a> AND
+              <li><a href="https://www.khronos.org/registry/webgl/extensions/EXT_texture_compression_rgtc/">EXT_texture_compression_rgtc</a>
+            </ul>)
+        </ul>
+    </p>
+    <div class="note rationale">
+        <p>
+            To best support our ecosystem, we require implementations to support either S3TC and RGTC formats (universal on desktop) or ETC2/EAC formats (universal on mobile).
+            This guarantees to authors that they can always use compressed textures (including srgb variants) on all devices while maintaining support for as few as two different formats.
+        </p>
+    </div>
+    <div class="note">
+        <p>
+          There are roughly equivalent formats in each suite for the following uses:
+          <table>
+            <tr>
+              <th>Usage</th>
+              <th>S3TC/RGTC option (desktop)</th>
+              <th>ETC2/EAC option (mobile)</th>
+            </tr><tr>
+              <td>R11 unsigned</td>
```

Comment:
It's a bit misleading to call the usage "R11" for both options here. In the EAC enum name, "11" refers to the format's implementation detail that it uses 11 bits of precision during interpolation. This is not applicable to RGTC.

Same comment for the next three entries.

Reply:
I think it's "usefully mostly true" (and it's indeed non-normative) in that it imparts the right instinct for how the format behaves.
The spec says:
> We will now go into detail how the decoding is done. The result will be an
11-bit fixed point number where 0 represents 0.0 and 2047 represents 1.0. This is
the exact representation for the decoded value. However, some implementations
may use, e.g., 16-bits of accuracy for filtering.

It's as true as the RGB8 compressed texture formats, which also aren't *really* 8-bit.
I think it's a useful approximation to make, even if it may not be fully true.

We can see RGTC has the same block-size requirements as its EAC counterparts, so I expect similar there. Reading through the decoding in the RGTC extension, we can see that max precision there is 8+3=11 bits as well.

Reply:
RGTC (and S3TC, btw) precision is very loosely defined (and varies between GPU vendors), so any specific number is somewhat speculative. Assuming that readers will get only the "instinct" from this table, let's keep it this way.